ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/checkpointsync/CheckpointSyncChainDownloader.java

```diff
@@ -11,8 +11,7 @@
  * specific language governing permissions and limitations under the License.
  *
  * SPDX-License-Identifier: Apache-2.0
- */ package org.hyperledger.besu.ethereum.eth.sync.checkpointsync;
```

Comment:
think you lost the package line here

Reply:
spotless apply bug it seems ... let me manually fix it.